app/views/activities/index.html.slim

```diff
@@ -1,7 +1,7 @@
 h1.header
   = icon('comments-alt')
   span Activities
-  = link_to '<span>Atom Feed</span>'.html_safe, activities_path(format: :atom), class: 'atom'
+  = link_to '<span>Atom Feed</span>'.html_safe, activities_path(team_id: params[:team_id], format: :atom), class: 'atom'
```

Comment:
I tried that already, this didn't seem neat to me. I see what you mean, but I didn't get the results I needed. I'll look into it again. 
(We are still working on this.)